.pipelines/templates/.builder-release-template.yaml

```diff
@@ -176,6 +176,16 @@ steps:
       Contents: 'trivy-images-table.txt'
       TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
+  - task: CopyFiles@2
+    condition: and(eq(variables.OS_SKU, 'Ubuntu'), eq(variables.ARCHITECTURE, 'X86_64'), in(variables.OS_VERSION, '22.04', '24.04'), eq(variables.FEATURE_FLAGS, 'None'))
+    displayName: Copy CIS Reports
+    inputs:
+      SourceFolder: '$(System.DefaultWorkingDirectory)'
+      Contents: |
+        cis-report.html
+        cis-report.txt
+      TargetFolder: '$(Build.ArtifactStagingDirectory)'
```

Comment:
Do these artifacts get auto-published to the release notes folder during release? I hope not. I'd rather we keep these reports internal for now - not because there's anything to hide but because I can't guarantee the stability of the format at this time.

Reply:
no, we specifically select the *-release-notes and *image-bom artifacts when building the release notes PRs, these won't be included even if they're now included in build artifacts